core/internal/cluster/kafka_cluster.go

```diff
@@ -120,14 +138,29 @@ func (module *KafkaCluster) mainLoop(client helpers.SaramaClient) {
 	defer module.running.Done()
 
 	for {
-		select {
-		case <-module.offsetTicker.C:
-			module.getOffsets(client)
-		case <-module.metadataTicker.C:
-			// Update metadata on next offset fetch
-			module.fetchMetadata = true
-		case <-module.quitChannel:
-			return
+		// NOTE: or I could simple create a very long tick ¯\_(ツ)_/¯
+		if module.groupsReaperRefresh != 0 {
```

Comment:
Maybe just stop the ticker if `groupsReaperRefresh == 0`?

Reply:
yes good idea, I didn't know that calling `ticker.Stop()` will *not* close the channel, so it'll make things easier here